cmd/nodepool/none/create.go

```diff
@@ -0,0 +1,44 @@
+package none
```

Comment:
I don’t think is clear just yet how we'll angle None. We could communicate that when platform None we support NodePool to generate an ign config for you and enable at the cli, though that feels weird since that NodePool is not scalable nor have any capability.
Please drop it for now until we agree that we need it.

Reply:
Dropped None platform